rails/spec/requests/api/public_stories_spec.rb

```diff
@@ -83,6 +86,38 @@
       )
     end
 
+    # Story:
+    # - place 1
+    # - speaker - will be deleted in test
+    # - language: English
+    let!(:story_with_no_speaker) do
+      create(
+        :story,
+        community: orphan_community,
+        topic: "test work",
+        language: "English",
+        places: [place_1],
+        speakers: [speaker_being_deleted],
+        permission_level: :anonymous
+      )
+    end
+
+    # Story:
+    # - place - will be deleted in test
+    # - speaker 1
+    # - language: English
+    let!(:story_with_no_place) do
+      create(
+        :story,
+        community: orphan_community,
+        topic: "test work",
+        language: "English",
+        places: [place_being_deleted],
+        speakers: [speaker_1],
+        permission_level: :anonymous
+      )
+    end
```

Comment:
I'd rather utilize the existing story/place/speaker setup rather than adding new ones. I know it makes it easier to conceptually read "place_being_deleted" and the like, but it does slow down the tests considerably since we are forcefully creating and recreating all of these stories, places, and speakers for each test.